src/main/java/core/basesyntax/figure/RightTriangle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.figure;
+
+public class RightTriangle extends AbstractFigure implements Figure {
+
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(String color, double firstLeg, double secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double getArea() {
+        return 0.5 * firstLeg * secondLeg;
+    }
+
+    @Override
+    public void draw() {
```

Comment:
The method 'draw' should not be placed in this class as it is conceptually different from 'getArea'. It's better to separate these concerns into different interfaces or classes.